github/issues.go

```diff
@@ -238,6 +238,10 @@ func (s *IssuesService) Get(owner string, repo string, number int) (*Issue, *Res
 //
 // GitHub API docs: http://developer.github.com/v3/issues/#create-an-issue
```

Comment:
Both `IssuesService.CreateComment` and `PullRequestsService.CreateComment` have a new "Comments longer than 65536 will return an error." sentence in the comment mentioning the limit. Why not here?